utils/rpms/daos.spec

```
@@ -3,7 +3,7 @@
 
 %define daoshome %{_exec_prefix}/lib/%{name}
 
-%global spdk_max_version 18.07
+%global spdk_max_version 19.04.1
```

But we don't know this yet do we?  If/when 19.04.2 comes out we don't want to have to update here to get it.

Or 19.05 or 19.45, or 20.0, etc. assuming they are all functionally equivalent to the feature set that we are using from 19.04.1.

The point here is to only update this value when we know of a version that has been released that is incompatible.

So for now, I would say just so as to not have to remove the various references to the value throughout the file, just set it to some very high (1000 perhaps) value.

In all honesty this should probably be the whole expression:
```suggestion
%global spdk_max_version "<= 19.04.1"
```

and then substituting the various `<= %{spdk_max_version}`s in this file with `%{?spdk_max_version}`.  But I'm not asking you to do this in this patch.  Just set the high value and I will do the suggested change at some later point.